template-only-bin/install-template.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
What's the suggested way of running this script? Do I clone the repo and then execute it from the root of the repo?

Reply:
I was thinking something like `curl https://raw.github.com/.../install-template.sh | bash -s`. Does that feel too scary to run a bash script from the internet

Reply:
`npm install` probably does worse so seems fine to me 😄  – I think it'd be helpful to add that instruction to the README if this is the preferred way of setting things up after this is merged.

Reply:
Sounds good, I was debating whether to do that in a later PR since this link won't resolve until the script is merged to main – https://raw.githubusercontent.com/navapbc/template-infra/main/template-only-bin/install-template.sh – but I figure it's easy enough to fix if I get it wrong, so I'll just add the instructions now